### PR TITLE
Clean up docs folder

### PR DESCRIPTION
**Pull request by llzmb**
Changes:
- Delete obsolete files.
- Merge files.
- Rename files.
- Fix references.
- Fix typos.
- Fix line length.

### CODE REVIEW

File: docs/fuzzing_binary-only_targets.md [hunk 1]
```diff
@@ -1,83 +1,293 @@
...
-Note that several other binary rewriters exist, all with their advantages and
-caveats.
+Note that there is also honggfuzz:
+[https://github.com/google/honggfuzz](https://github.com/google/honggfuzz) which
+now has a qemu_mode, but its performance is just 1.5% ...
 
-### Frida
+If you like to code a customized fuzzer without much work, we highly recommend
+to check out our sister project libafl which will support QEMU, too:
+[https://github.com/AFLplusplus/LibAFL](https://github.com/AFLplusplus/LibAFL)
```
Comment (llzmb) [inline docs/fuzzing_binary-only_targets.md]:
```suggestion
If you like to code a customized fuzzer without much work, we highly recommend
to check out our sister project libafl which supports QEMU, too:
[https://github.com/AFLplusplus/LibAFL](https://github.com/AFLplusplus/LibAFL)
```

I looked at the [repo](https://github.com/AFLplusplus/LibAFL). It seems like by now it _does_.

Comment (vanhauser-thc) [reply docs/fuzzing_binary-only_targets.md]: yes it does now :)

File: README.md [hunk 2]
```diff
@@ -18,129 +18,147 @@ AFL++ is maintained by:
...
-   If the program takes input from a file, you can put `@@` in the program's 
+   If the program takes input from a file, you can put `@@` in the program's
    command line; AFL will put an auto-generated file name in there for you.
 
-4. Investigate anything shown in red in the fuzzer UI by promptly consulting 
-   [docs/status_screen.md](docs/status_screen.md).
+4. Investigate anything shown in red in the fuzzer UI by promptly consulting
+   [docs/afl-fuzz_approach.md#understanding-the-status-screen](docs/afl-fuzz_approach.md#understanding-the-status-screen).
+
+5. Interpret the output, see
```
Comment (vanhauser-thc) [inline README.md]: please remove 5. I dont think we need this here

**Comment by vanhauser-thc** [comment]
for me it is just the tiny nit, otherwise it looks great :)

**Comment by llzmb** [comment]
@vanhauser-thc, thank you for your feedback. Done! :)

**Pull request merged**